### PR TITLE
Harmonize glyphs/variants of three non-slavic Cyrillic letters.

### PR DESCRIPTION
**Pull request by jmcwilliams403**
Schwa top hook behavior:
`ОоӨөЭэӘә`
(not shown is real turned e which is unchanged).
Upright:
image
Italic:
image
Compared to slab:
Upright:
image
Italic:
image
Cyrillic Straight U behavior:
All Capital U variants:
`УЎҮҰ`
image
All Lower U variants:
`уўүұ`
image

### CODE REVIEW

**Comment by be5invis** [comment]
I don't think ogoneks are widely accepted for Ҙҙ Ҫҫ Ҿҿ. Let us just follow the Unicode chart.

File: packages/font-glyphs/src/letter/latin/c.ptl [hunk 1]
```diff
@@ -414,7 +414,5 @@ glyph-block Letter-Latin-C : begin
 
 	derive-composites 'CCedilla' 0xC7 'C' 'cedillaExtShapeBelowOArc'
 	derive-composites 'cCedilla' 0xE7 'c' 'cedillaExtShapeBelowOArc'
-	derive-composites 'CCedillaAcute' 0x1E08 'C' 'cedillaExtShapeBelowOArc' 'acuteAbove'
-	derive-composites 'cCedillaAcute' 0x1E09 'c' 'cedillaExtShapeBelowOArc' 'acuteAbove'
-	derive-composites 'cyrl/The' 0x4AA 'cyrl/Es' 'cedillaExtShapeBelowOArc'
-	derive-composites 'cyrl/the' 0x4AB 'cyrl/es' 'cedillaExtShapeBelowOArc'
+	derive-composites 'cyrl/The' 0x4AA 'cyrl/Es' 'ogonekBelow'
```
Comment (be5invis) [inline packages/font-glyphs/src/letter/latin/c.ptl:417]: Please revert to use Cedilla. The Es-Ogonek form is only considered as "Occasionally used" per Wikipedia.

File: packages/font-glyphs/src/letter/greek/lower-epsilon.ptl [hunk 2]
```diff
@@ -490,8 +490,8 @@ glyph-block Letter-Greek-Lower-Epsilon : begin
 	select-variant 'cyrl/DzjeKomi' 0x506 (follow -- 'cyrl/Ksi')
 	select-variant 'cyrl/dzjeKomi' 0x507 (follow -- 'cyrl/ksi')
 
-	derive-composites 'cyrl/ZeCedilla' 0x498 'cyrl/Ze' 'cedillaExtShapeBelowOArc'
-	derive-composites 'cyrl/zeCedilla' 0x499 'cyrl/ze' 'cedillaExtShapeBelowSOArc'
+	derive-composites 'cyrl/Dhe' 0x498 'cyrl/Ze' 'ogonekBelow'
```
Comment (be5invis) [inline packages/font-glyphs/src/letter/greek/lower-epsilon.ptl:493]: Please revert to use Cedilla to be conformal to Unicode chart.

File: packages/font-glyphs/src/letter/latin/lower-e.ptl [hunk 3]
```diff
@@ -131,26 +131,25 @@ glyph-block Letter-Latin-Lower-E : begin
 			hookend O (sw -- stroke)
 			g4 (df.width - [xTerminalR df]) [HookHeight top stroke]
 
-	define [AbkCheShape] : with-params [fDesc Body df top tailSlab] : glyph-proc
+	define [AbkCheShape] : with-params [Body df top tailSlab] : glyph-proc
```
Comment (be5invis) [inline packages/font-glyphs/src/letter/latin/lower-e.ptl:134]: Please revert and use straight descender, to be conformal to Unicode chart.

**Review by be5invis — CHANGES_REQUESTED** [review]

**Comment by berrymot** [comment]
the unicode charts do not prescribe glyphs

**Comment by jmcwilliams403** [comment]
I kept some of the code cleanup and the `cyrl/Dhe` rename but otherwise the descender glyphs are back to normal.

**Review by be5invis — APPROVED** [review]

**Pull request merged**